torch/lib/c10d/ProcessGroupMPI.cpp

```diff
@@ -0,0 +1,281 @@
+#include "ProcessGroupMPI.hpp"
+
+#include <iostream>
+#include <map>
+#include "mpi-ext.h" // Needed for CUDA-sware check
```

Comment:
Should be `<mpi-ext.h>` since this is a dependency (sourced from the include path, not the current directory).

Also typo `sware` -> `aware`.

Reply:
that's right. surprising that the compiler didn't complain. pretty sure I was using coda-aware mpi